libs/platform/src/lib/components/form/combobox/combobox/combobox.component.html

```diff
@@ -96,7 +96,7 @@
         <ng-container *ngIf="isGroup">
             <ng-container *ngFor="let group of _suggestions">
                 <ng-container *ngIf="!groupItemTemplate">
-                    <li fd-list-group-header role="group" tabindex="0">
+                    <li fd-list-group-header role="option" tabindex="0">
```

Comment:
@InnaAtanasova are we sure that this element needs to be tab-able to be compliant with accessibility requirements? I think just using roles to have the screenreader read it out may be sufficient

Reply:
if we use role="group" screenreader will not read it

Reply:
I'm saying, can we remove `tabindex="0"` ?

Reply:
tabindex was removed